src/components/Datatable/ToolsTabItem/ToolsTabItem.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { isNotUndefined, noop } from 'ramda-adjunct';
+import styled from 'styled-components';
+
+import { SpacingSizeValuePropType } from '../../../types/spacing.types';
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconTypes, SSCIconNames } from '../../Icon/Icon.enums';
+import { SSCIcons, Types } from '../../Icon/Icon.types';
+import Button from '../../Button/Button';
+import StyledIcon from '../../Button/StyledIcon';
+import { ButtonSizes } from '../../Button/Button.enums';
+import { ToolsTabItemProps } from './ToolsTabItem.types';
+import { ToolsTabItemStates } from './ToolsTabItem.enum';
+
+const DefaultToolsTabItem = styled(Button)`
+  &&& {
+    padding: ${pxToRem(9, 12)};
+    color: ${getColor('graphite2B')};
+    &:focus:not(:disabled),
+    &:hover:not(:disabled),
+    &:not([href]):not([tabindex]):not(:disabled):hover,
+    &.hover,
+    &.focus {
+      color: ${getColor('graphite4B')};
+      svg {
+        color: ${getColor('graphite2B')};
+      }
+    }
+  }
+`;
+
+const ActiveToolsTabItem = styled(Button)`
```

Comment:
Those styled should be probably inherited from the `DefaultToolsTabItem` not from the base `Button` component